util/tracing/tracer.go

```diff
@@ -72,6 +72,7 @@ func JoinOrNewSnowball(opName string, carrier *Span, callback func(sp basictrace
 
 func defaultOptions(recorder func(basictracer.RawSpan)) basictracer.Options {
 	opts := basictracer.DefaultOptions()
+	opts.ShouldSample = func(traceID int64) bool { return false }
```

Comment:
leaving the unused named parameter was actually intentional. Without it, this is a lot let clear and would probably need to be documented. Right now, it's pretty obvious even without documentation that we're saying "it doesn't matter which traceID you give me, never sample."